config/direct.js

```diff
@@ -0,0 +1,11 @@
+import { createOneNote } from "../modules/notes/controllers/createNotes";
```

Comment:
1. Move this file to the `notes/controllets/index.js`
2. `deleteOneNote`. Check all the counts in naming